docs/docs/product-guide/05-time-tracking.md

```
@@ -35,8 +49,8 @@ The Month view of your timesheet provides an efficient way to view all hours log
 
 Here, you can easily track your total hours worked for the month.
 
-### **Adding Time in the Monthly Overview**
+### Adding Time in the Monthly Overview
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)